doxygen/doxygen.config.in

```diff
@@ -145,8 +145,8 @@ REFERENCES_LINK_SOURCE = YES
 SOURCE_TOOLTIPS        = YES
 USE_HTAGS              = NO
 VERBATIM_HEADERS       = YES
-CLANG_ASSISTED_PARSING = NO
-CLANG_OPTIONS          = 
+#CLANG_ASSISTED_PARSING = NO
```

Comment:
Why do we need to comment `CLANG_ASSISTED_PARSING` and `CLANG_OPTIONS` out?

Reply:
We don't really, I commented them out because they were giving me warnings when running Doxygen. I can add them back in, I don't think it makes a difference beyond the warnings.